### PR TITLE
BigTable: Timestamp system test fix

### PR DESCRIPTION
**Pull request by chemelnucfin**

### CODE REVIEW

File: bigtable/tests/system.py [hunk 1]
```diff
@@ -311,8 +311,15 @@ def _write_to_row(self, row1=None, row2=None, row3=None, row4=None):
         timestamp1 = _datetime_from_microseconds(timestamp1_micros)
         # 1000 microseconds is a millisecond
         timestamp2 = timestamp1 + datetime.timedelta(microseconds=1000)
+        timestamp2_micros = _microseconds_from_datetime(timestamp2)
+        timestamp2_micros -= (timestamp1_micros % 1000)
```
Comment (jonparrott) [inline bigtable/tests/system.py]: Weird, are you truncating the nanosecond part?

Comment (chemelnucfin) [reply bigtable/tests/system.py]: I was just following what was done before.  IIRC, the tests work truncated or not.

Comment (jonparrott) [reply bigtable/tests/system.py]: so why is this needed?

Comment (chemelnucfin) [reply bigtable/tests/system.py]: I see your point.  Removed the truncations since they don't do anything anyway.

**Pull request merged**